Patches/Sergal/PawnKinds_Frog.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Sergals Gnolls Lions and Frogs</modName>
+			</li>
+
```

Comment:
The magazine counts here seem pretty high. Hopefully that's just indicative of a bow, musket, or some other form of 1-round weapon?

Reply:
All Frog pawns were only ever seen armed with bows and throwable javelins, or melee weapons, so these values are appropriate here.